### ISSUE TITLE
Query level of neo block in graphql  - Feature Request

### ISSUE THREAD

**Issue by sgtpenguin** [spicywebau/craft-neo#505]
I don't think it's currently possible to get the level of the neo block through graphql.  Would this be possible to add?  I want to be able to tell if a block is being used at the top level without constructing this logic in my front end code.

**Comment by ttempleton** [comment]
GraphQL Neo block `level` fields are available as of the latest version 2.11.13.

Level example:

```
neoField(level: 0) {
    ... on neoField_text_BlockType {
        level
        text
    }
}
```

Returned data example:

```
"neoField": [
    {
        "level": 1,
        "text": "Parent text block"
    },
    {
        "level": 2,
        "text": "Child text block"
    },
    {
        "level": 1,
        "text": "Another top-level text block"
    }
]
```

**Comment by sgtpenguin** [comment]
awesome.  thanks!
